src/main/java/com/esri/arcgisruntime/toolkit/skins/ScalebarSkin.java

```diff
@@ -193,7 +193,7 @@ protected HPos getAlignment() {
    */
   Region calculateRegion(Region region) {
     Group root = new Group();
-    Scene dummyScene = new Scene(root);
+    Scene unusedScene = new Scene(root);
```

Comment:
The inclusive language doc says to use "place holder" instead of "dummy".

Reply:
yes, i assume you are just stating a fact and you have no follow up comment...

Reply:
Sorry, I wasn't clear. I am saying it should be `placeHolderScene` not `unusedScene`.

Reply:
is something wrong with 'unusedScene' ?

Reply:
No, although it isn't actually unused as the method won't work without it. I just think we are better following the guidance we have been given on what to replace and what to replace it with.